### PR TITLE
docs: incorporate pfe-codeblock on storybook pages

### PR DESCRIPTION
**Pull request by castastrophe**
- Updates all storybook demos to use the pfe-codeblock component for rendering
- Adds a decode HTML function for converting user-added HTML into renderable output
- Update pfe-primary-detail storybook

### Preview
- [Preview instance](https://deploy-preview-1308--happy-galileo-ea79c4.netlify.app/storybook/)

### Related issue

- (#1305) pfe-primary-detail storybook

### Ready-for-merge Checklist

Check off items as they are completed.  Feel free to delete items if they are not applicable.

- [x] Expected files: all files in this pull request are related to one request or issue (no stragglers or scope-creep).
- [x] Repository compiles and tests pass.

### Merging

Please **squash** when merging and ensure your commit message uses [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/#summary) formatting.

**Be sure to share your updates with the [patternfly-elements-contribute@redhat.com](mailto:patternfly-elements-contribute@redhat.com) mailing list!**

### CODE REVIEW

File: elements/pfe-primary-detail/README.md [hunk 1]
```diff
@@ -67,10 +67,6 @@ For this component to work, there should be an equal number of `details-nav` and
 - `details-nav--header`: In case content needs to be added at the top of the nav area. Will not be matched up with `details` content.
 - `details-nav--footer`: In case contnet needs to be added at the bottom of the nav. Will not be matched up with `details` content.
 
-## Attributes
```
Comment (castastrophe) [inline elements/pfe-primary-detail/README.md:70]: @wesruv I didn't see this attribute in the JS code for the component - is it okay if I remove this from the docs?

File: elements/pfe-primary-detail/demo/pfe-primary-detail.story.js [hunk 2]
```diff
@@ -1,14 +1,6 @@
...
-// !!
-// !! Having a hard time understanding how to get this setup,
-// !! leaving it with a working sample that does not respond to
-// !! story book configuration.
-// !!
-// !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
-
 import { storiesOf } from "@storybook/polymer";
-import * as storybookBridge from "@storybook/addon-knobs";
+import * as bridge from "@storybook/addon-knobs";
```
Comment (castastrophe) [inline elements/pfe-primary-detail/demo/pfe-primary-detail.story.js:2]: @wesruv Would love you review of this storybook update when you get a chance - I updated it when I was working on the codeblock injection

**Comment by castastrophe** [comment]
@alistairmc When you get a chance, I'd love your review on this PR that brings in pfe-codeblock on the storybook demo pages.  I want to make sure I implemented it with the right settings and that you think it looks good in this context.  Thanks!

**Comment by alistairmc** [comment]
@castastrophe This looks great to me, really good to see the code-block used in the wild!.
I would be happy except for I see the visual regression tests are not passing have you updated the base images?

**Comment by castastrophe** [comment]
> @castastrophe This looks great to me, really good to see the code-block used in the wild!.
> I would be happy except for I see the visual regression tests are not passing have you updated the base images?

We don't snapshot the storybook pages so we shouldn't need to update any baseline images. Thank you!  I'm excited to see it on our site!

**Review by alistairmc — COMMENTED** [review]
This looks good to me, following the message about no need for visual regression tests.

**Comment by castastrophe** [comment]
@alistairmc Awesome!  Thank you so much!  Would you mind approving when you get a chance?

File: elements/pfe-primary-detail/README.md (hunk 1 above)
Comment (wesruv) [reply elements/pfe-primary-detail/README.md:70]: It's controlled with CSS, no JS necessary. Should stay in, could change it to a class if it's better suited as a class?

**Review by wesruv — CHANGES_REQUESTED** [review]
Vast improvement on the storybook front, thank you! Just want to keep the docs for consistent-height.

File: elements/pfe-primary-detail/README.md (hunk 1 above)
Comment (castastrophe) [reply elements/pfe-primary-detail/README.md:70]: Oh!  I looked and didn't see it in the CSS.  An attribute is great, my bad for missing how it was being implemented.

Comment (castastrophe) [reply elements/pfe-primary-detail/README.md]: Just pushed up the update reverting this change

**Review by wesruv — APPROVED** [review]
Long Grifting Those Marsupials

**Pull request merged**